redirects.map

```diff
@@ -17,3 +12,5 @@
 /2.2/en/installconfig-nodes-deploy /2.2/en/nodes-deploy;
 /2.2/en/installconfig-nodes-power-types /2.2/en/nodes-power-types;
 /2.2/en/intel-rsd /2.2/en/nodes-comp-hw;
+~^/(?<version>2.5|2.4|2.3|2.2|2.1|2.0|1.9)(/|/en|/index|/en/index)?$ /${version}/en/;
+~^(/|/devel|/2.6|/devel/en|/2.6/en)(?<page>(/.*)?)$ https://maas.io/docs${page};
```

Comment:
```
curl -I http://127.0.0.1:8996/2.6/en   ->  Location: https://maas.io/docs/en
curl -I http://127.0.0.1:8996/2.6/en/   ->  Location: https://maas.io/docs/en/
curl -I http://127.0.0.1:8996/devel/en   ->  Location: https://maas.io/docs/en/
curl -I http://127.0.0.1:8996/devel/en/   ->  Location: https://maas.io/docs/en/
curl -I http://127.0.0.1:8996/devel/en/somewhere   -> https://maas.io/docs/en/somewhere

```

Reply:
Yeah so this is fine because `/docs/2.6/en` currently redirects to `docs.maas.io/2.6/en`, which works fine.

And then when this work goes live, the URL at `/docs/2.6/en` will hit the redirects in this PR: https://github.com/canonical-web-and-design/maas.io/pull/349/files - which will then redirect that to `/docs`.

Reply:
As in, the idea is that maas.io will ultimately be responsible for the choosing where to send `/2.6`, `/2.5` etc. This is important because ultimately we will naively redirect `https://docs.maas.io/*` -> `https://maas.io/docs/*` in Kubernetes.

Reply:
Updated QA steps.

Reply:
Actually I've now fixed it so my original QA steps work =) sorry.